packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -30,15 +30,25 @@ interface WatchPR {
   url: string;
 }
 
-handler.listPRs = async function listPRs(): Promise<WatchPR[]> {
+handler.getDatastore = async function getDatastore() {
   const query = datastore.createQuery(TABLE).order('created');
   const [prs] = await datastore.runQuery(query);
+  return [prs];
+};
+
+handler.listPRs = async function listPRs(): Promise<WatchPR[]> {
```

Comment:
For each of these, would love a header with a 1-2 sentence description of what the function is used for.  Helpful for unsuspecting souls that wonder into the codebase later (me :))